Gemfile

```diff
@@ -40,7 +34,6 @@ gem 'simple_form', '3.0.1'
 gem 'facebox-rails'
 gem 'strong_presenter', '~> 0.2.2'
 gem 'render_anywhere'
-gem 'pygments.rb', '0.5.4'
```

Comment:
We use pygments.rb in the app (to do syntax highlighting on submissions), so we can't remove this dependency.

Reply:
yep you're right - did the agressive thing to start with, but i've resolved that now and removed the WIP: tag, so I think this is good review :)